test_graphs.py

```diff
@@ -33,4 +37,7 @@ def pytest_train_model(model_type):
     error, true_values, predicted_values = test_trained_model(tmp_file, model_type)
 
     for true_value, predicted_value in zip(true_values, predicted_values):
-        assert abs(true_value[0] - predicted_value[0]) < 1e-3
+        if model_type == "PIN" or model_type == "MFC":
+            assert abs(true_value[0] - predicted_value[0]) < 0.1
```

Comment:
Based on running these locally I think we could tighten these tolerances a little (but until it's clear why MFC fails when run separately I'm not sure)

Reply:
The problem is as small as it can be. The structure has 4 atoms. Smaller than this makes no sense. Bigger than this, the structure becomes combinatorially complex, so the problem becomes fo a comparable challenge as the DFT datasets. 
Should we relax the tolerance to 0.5 for all the models?
Otherwise, I have no immediate solutions to the task of developing the unit test.

Reply:
GIN, GAT, and MFC use aggregations that can ambiguously identify two different graphs as equivalent. 
PIN mixes and simultaneously uses the aggregations used by the previous thee models to make sure that different graphs are properly recognized as different. This is one way to explain why PIN does not show up in any of the inconsistencies we are dealing with now.

Reply:
I was only thinking of test accuracy here (same as Pei), not test size. Based on running locally, I think we could use 0.05 for PNN, 0.2 for MFC, and 0.4 for GIN/GAT. But maybe I'm trying to be too aggressive.

Reply:
With your last comment, we could just use 0.1 for PNN and 0.4 for the rest and try to avoid unnecessary failures